### ISSUE TITLE
Google Compliance

### ISSUE THREAD

**Issue by TheLastProject** [TheLastProject/Catima#439]
![Action Required Your app is not compliant with Google Play Policies (Catima — The Libre Card Wallet)](https://user-images.githubusercontent.com/1885159/135721623-9df9ad54-81f3-4dac-8410-658413e47b9e.png)

I have no clue what Google is smoking this time, this seems like complete bullshit, and we've been using this title for years.

Looking at Google Play there are other apps using very similar titles like: https://play.google.com/store/apps/details?id=com.innovativeiteration.rewardsapp

"Reward Cards : The Card Wallet"

Why is "Catima - The Libre Card Wallet" not okay when "Reward Cards : The Card Wallet" is?

**Comment by TheLastProject** [comment]
I've filed an appeal, stating I have insufficient information:

I do not understand how the title is supposedly breaking any of the policies. We are not trying to manipulate the placement, just make clear what the app is. Alternatives like Stocard also put similar text in the app name: "Stocard - Rewards Cards Wallet" to make clear that it's a loyalty card app.

Please explain in more detail which part of the title you claim is violating the policy and why this only relates to the Norwegian and Dutch titles, given they are literally just translations of the original English title.

**Comment by TheLastProject** [comment]
Appeal ID 6-5923000031473.

**Comment by TheLastProject** [comment]
The Norwegian and Dutch titles haven't even changed since 5 months!

https://github.com/TheLastProject/Catima/blob/48f680f7bd4ad6bb5974a94811a2d796d56dddbe/fastlane/metadata/android/nb-NO/title.txt
https://github.com/TheLastProject/Catima/blob/48f680f7bd4ad6bb5974a94811a2d796d56dddbe/fastlane/metadata/android/nl-NL/title.txt

**Comment by TheLastProject** [comment]
As I kinda guessed, it was thrown into Google translate and flagged because it translated to the English "free", ignoring it doesn't mean "at no cost":

For example, your app's title in nl_NL and no_NO locales currently contains text that indicates price and promotional information (E.g. "free", "vrije", "frie"). You can refer to the attached screenshot for additional information.

**Comment by TheLastProject** [comment]
I have responded:

Vrij is not Dutch for "at no cost", it is Dutch for libre -> open source. The same goes for Norwegian.

I repeat: neither translations talk about price.

**Comment by TheLastProject** [comment]
Got a response:

Hi Sylvia,

Thanks for your patience.

As mentioned in previous email, **your app's title in nl_NL and no_NO locales currently contains text that indicates price and promotional information.**

Please note that the examples provided here are not exhaustive. We recommend that you review all translations of your store listing to comply with **User Ratings, Reviews and Installs** policy.

You can read through the User Ratings, Reviews and Installs policy page for more details and examples of common violations.

Please update your app to fix **this issue**. You may also want to double check that your app complies with all other Developer Program Policies, as additional enforcement could occur if there are further policy violations.

Thanks for your understanding and continued support.

**Comment by TheLastProject** [comment]
So, of course, this doesn't answer the question. So I asked the same question again:

As I said, none of the translations talk about price. While "free" may be ambiguous in English, it is not in Dutch or Norwegian. And for English we use libre to disambiguate.

Is stating the app is Open Source considered "promotional information"? Because you keep talking about price but I already stated that there is no price info in any of the titles.

**Comment by TacoTheDank** [comment]
> I have responded:
> 
> Vrij is not Dutch for "at no cost", it is Dutch for libre -> open source. The same goes for Norwegian.
> 
> I repeat: neither translations talk about price.

Lmao so basically Google's translation service is subpar. What a surprise!

I guess if nothing else can be done, perhaps the "Libre" could be removed from the Dutch and Norwegian translations? It's not ideal, but at least it'll allow updates to be published again.

Or, perhaps Catima's Play Store name could be changed to something like "Catima - Open Source Card Wallet" across all translations? I think a change like this would greatly reduce the chances of this happening again, as "open source" is probably much less debatable from a translation standpoint than "libre" is.

Anyhow, hope this is resolved soon :)

**Comment by TheLastProject** [comment]
Unbelievable, Google still insists I'm not compliant, copy-pasting the same message despite me stating we're not talking price.

Hi Sylvia,

Thanks for your reply.

As much as I'd like to help, I'm not able to provide any more detail or a better answer to your question. **In our previous email, I made sure to include all the information available to me**.

As mentioned previously, **your app's title in nl_NL and no_NO locales currently contains text that indicates price and promotional information.**

You can read through the User Ratings, Reviews and Installspolicy page and **refer to our previous emails for more details**.

Please update your app to fix **this issue**. You may also want to double check that your app complies with all other Developer Program Policies, as additional enforcement could occur if there are further policy violations.

Thanks for your understanding and continued support.

**Comment by TheLastProject** [comment]
Whatever Google, you win: ab7505c67adba1a040c8324d701db1d12b6a3a66

And I'm sick of that "Thanks for your understanding and continued support" bullshit.

So, one last email reply:

But the Dutch and Norwegian app titles do NOT contain price info!

How is it possible you can insist I am not compliant while I have provided an explanation that your statement is wrong!

But fine, you win. I don't care to waste more time on this. I have edited the Dutch and Norwegian titles.

Thanks for your lack of understanding and continued support.

**Comment by TheLastProject** [comment]
Fuck you, Google. Fuck you.

**Comment by TheLastProject** [comment]
Fixed in ddccbad0202aaa88a567ee5bc931840ed0231331 hopefully. Who the fuck knows with these incompetent fucks.

**Comment by TheLastProject** [comment]
![image](https://user-images.githubusercontent.com/1885159/136180398-e6051aa6-4765-4aac-bc54-b6a2cdadd8c0.png)

OH COME ON

**Comment by TheLastProject** [comment]
017034a78804eb092917086376192796587e373e ...

**Comment by TheLastProject** [comment]
![image](https://user-images.githubusercontent.com/1885159/136434121-02f7a4f4-1729-4587-8b6e-8be3360c0290.png)

Would you look at that, using "Open Source" instead of words that unambiguously mean "free as in freedom" and not "free as in price" and it's all accepted.

So yes, this 100% confirms it, I got yelled at by Google for using translations of the word "libre", probably because throwing it in Google Play returns "free" even though the languages in questions were not ambiguous like English is.

Good job Google, too incompetent to use a dictionary.

**Comment by TheLastProject** [comment]
Google managed to find 2 more I guess...

![image](https://user-images.githubusercontent.com/1885159/140650588-0be275bd-7cda-46ed-893e-ecef5ccb2ccd.png)

**Comment by TheLastProject** [comment]
@pxn @70h Google marked your translations as "not okay". Any chance you could perhaps translate it some other way to avoid the word "libre" as Google confuses it for "at no cost" and price info "is not allowed"?

**Comment by pxn** [comment]
@TheLastProject removed "libre" from all cs_CZ translation, not only from title.txt

**Comment by TheLastProject** [comment]
Thank you for your quick response! It's sad Google is being this silly but as they control the Google Play Store we have no option but to play by their weird rules.

**Comment by TheLastProject** [comment]
@ersen0 Google sadly declined "Catima — Özgür Kart Cüzdanı" as Turkish title. From a quick web search it is clear that Özgür means free as in "independent", not "at no cost", but it's impossible to get Google to fix their review system sadly, so I am going to have to ask you to change the title please :(

**Comment by SaphireLattice** [comment]
Sounds like they just google translate the title and ban if it has "free" in the resulting english, rather than bothering to check the actual meaning and which "free" it was...

**Comment by TheLastProject** [comment]
> Sounds like they just google translate the title and ban if it has "free" in the resulting english, rather than bothering to check the actual meaning and which "free" it was...

Yup, they do exactly that... :(

**Comment by ersen0** [comment]
@TheLastProject it is really hard to come up with something meaningful with 30 character limit. I have changed it to "Catima — Özgürce Kart Cüzdanı" which means "freely card wallet". I am not sure if it makes any sense or it is compatible with google's nonsense.

**Comment by TheLastProject** [comment]
@ersen0 Thanks for your quick update! If you can't think of anything good you can also just use "Catima" without any subtitle or so. I will push the translation update to Google and we will see what they say :)

**Comment by TheLastProject** [comment]
We have been approved so closing this issue again until probably soon, knowing Google's incompetence

**Comment by TheLastProject** [comment]
![Screenshot_20211120-005926_K-9_Mail](https://user-images.githubusercontent.com/1885159/142705625-64eaa677-f78c-4f5b-a8a9-8b2e79a9ad90.png)

Hey @laralem, can you please change the portuguese translation? Google is still incompetent.

**Comment by TheLastProject** [comment]
@pxn It was fine for a while, but suddenly Google decided to disallow the Czech translation again... Sorry :(

 We found an issue in the following area(s):

    Title (cs_CZ): “Catima - Svobodná brašna karet”
    Title (pt_PT): “Catima gestor de cartões livre”

**Comment by TheLastProject** [comment]
The insanity has reached a new high, now I'm being punished for translations that literally do not exist... Google, WTF!

![photo5828036292007540777](https://user-images.githubusercontent.com/1885159/142989964-b7fb84c0-c053-4d58-81f3-a4a7c5d06119.jpg)

![Screenshot 2021-11-23 at 09 12 40](https://user-images.githubusercontent.com/1885159/142989934-d05f6fc0-2dfd-4955-934b-780501213019.png)
![Screenshot 2021-11-23 at 09 12 47](https://user-images.githubusercontent.com/1885159/142989953-672361c8-a7b2-4827-a932-3a164f4fec6e.png)
![Screenshot 2021-11-23 at 09 12 55](https://user-images.githubusercontent.com/1885159/142989955-d5cd3aeb-4bb5-416f-800c-816711f6419d.png)

**Comment by pxn** [comment]
@TheLastProject I didn't like that translation either, would reverting app name to untranslated work for You and Google?
Checked app store and i see other apps not translating too.

**Comment by TheLastProject** [comment]
@pxn It's also fine to just call it "Catima" without any subtitle or so, whatever works.

At this point I am seriously considering changing the source tagline, maybe something like "Private Card Wallet" or "Secure Card Wallet". This Google nonsense has been taking way too much time for me.

**Comment by IllusiveMan196** [comment]
What's funny is there are a ton of apps with literal English "free" in title. With it being not "free as freedom" but "free beer, yay!" so... yeah. Policies are weird. Edited my title version, so shouldn't be an issue at least for now.

**Comment by TheLastProject** [comment]
Oh, they hadn't even complained about the Ukranian version (but weirdly enough got upset about the months-ago-submitted Bulgarian version today). Thank you though :)

**Comment by justjanne** [comment]
Suggestion: One potential alternative would be using the acronym FLOSS in the title, as Google Translate either refuses to translate it, or translates it as floss as in the dental product.

**Comment by comradekingu** [comment]
@justjanne The point is to communicate in a way that is unambiguous and meaningful.
OSS, FOSS, or now FLOSS is the direct opposite, it doesn't have a related meaning, and is encapsulated in many layers of conflicting concepts to that end.
It communicates ~_something is meant to mean something here_, and it doesn't grant enough search-hits to include it, over how much noise it brings.

**Comment by TheLastProject** [comment]
Google also recently lowered the character count to 30 characters and we're nudging pretty close to that limit. At this point I just care about not wasting my time arguing with Google for hours when I could be making the app better instead, so I'm not planning any changes.

**Comment by justjanne** [comment]
@comradekingu Oh I agree it’s not as meaningful and not nearly as good as the "libre" option, but it seems to be the closest option that conveys the entire meaning and won’t get banned by Google.

I’ve faced similar issues with my own app, Quasseldroid, in the past, not just in the title but even in random translation strings, e.g. the about screen of the app.

**Comment by TheLastProject** [comment]
> but even in random translation strings, e.g. the about screen of the app

Oh am I not looking forward to that nonsense... :(

**Comment by comradekingu** [comment]
@justjanne FLOSSA contains a lot, but doesn't convey it.
It operates entirely in the capacity of losing what little recognition it had once translated.
Ideally speaking, Google should not dictate the language we use, even haphazardously.
To bend to their whims after rightfully fighting it down to the specifics of their madness would be a kneefall, and that is not a good message to send.
The otherwise good Quasseldroid website has a "Free & Open Source" (which is the wrong use of an ampersand) all the way down at the bottom. I would try to help, but it uses the enemy translation platform, Transifex.

**Comment by SpitFire-666** [comment]
[Leaving a comment here coz I can't comment on your blog post]

After months of effort as a beginner android dev, I built my first app. I did NOT find it easy 🤓. But then I went to publish it...

The whole process of publishing my app on Google Play was horrible for a regular joe like me. I still don't understand it all. Where's the "Just publish my damn app to the entire planet already" button???

Anyways I found the entire experience really discouraging. Even once it was available in the store, it was buried under millions of unrelated/sketchy apps, even when searching! Zero discoverability. 

Let's just say I won't be quitting my day job 😁
